### ISSUE TITLE
Another tagged release for Upsilon?

### ISSUE THREAD

**Issue by imadnyc** [UpsilonNumworks/Upsilon#360]
Upsilon seems to be an active project, yet it's latest tagged release is older than Omega's latest  tag. Is it possible for a stable tag to be created? I want to package this for NixOS, but there's no point if it's older than Omega - I'll just package that. Also, what is the release currently on the web app installer?

**Comment by PetitFrapo** [comment]
Upsilon is newer. Basically Upsilon has 3 release channels: stable, beta and dev. Dev is built every commit and has no themes because that would take far too much resources for something updated very often. Beta is updated every couple of commits when they are stable, and Stable is rarely updated. The latest dev build is a week old, and is stable, but beta is great too if you want themes. Lmk if I didn't answer your question.

**Comment by BloomyInDev** [comment]
> Upsilon seems to be an active project, yet it's latest tagged release is older than Omega's latest tag. Is it possible for a stable tag to be created? I want to package this for NixOS, but there's no point if it's older than Omega - I'll just package that. Also, what is the release currently on the web app installer?

Upsilon don't use release from GitHub. Instead, they post theirs releases on [Upsilon's website](https://getupsilon.web.app).
I think you will need to build it yourself if you want a specific version

**Comment by imadnyc** [comment]
@PetitFrapo @BloomyInDev So it seems that I should just build from beta?

**Comment by BloomyInDev** [comment]
> @PetitFrapo @BloomyInDev So it seems that I should just build from beta? 

Or build from dev

**Comment by imadnyc** [comment]
Sorry if this is a dumb question, but where can I find the beta branch? https://github.com/UpsilonNumworks/Upsilon/branches/all only seems to have an omega-beta branch, which was updated 2 years ago.

**Comment by BloomyInDev** [comment]
Beta files are on [Upsilon's website](https://getupsilon.web.app), you just have to scroll
They don't seem to keep a beta branch nor a "stable" branch (idk for the second)

**Comment by Yaya-Cout** [comment]
The betas are there : https://github.com/lolocomotive/Upsilon/tree/upsilon-beta, but it should be improved. I don't remember if there is a stable branch. Almost everything is in the `upsilon-dev` branch, which is almost as stable as others branches, so you could just use this one. On AUR (for Arch Linux), that's what the package does.
I don't remember if Nix require package to be reproducible, but if not, you could just use the `upsilon-dev` branch as source (and store the commit hash, if you want, but it's already saved automatically).
The spirit of Upsilon is more "rolling release".

**Comment by imadnyc** [comment]
Yeah, I wanted to target the beta so that I would have a stable-ish thing to build off of. Thanks!
